app/js/components/common/Header.js

```diff
@@ -5,119 +5,119 @@ import {ApiHelper} from '../../helpers/apiHelper';
-            this.setState({locationTags: response.results});
-            this.setState({defaultLocation: response.results[0].display});
-        });
+  constructor() {
+    super();
+    this.state = {
+      locationTags: [],
+      currentLocationTag: "",
+      defaultLocation: "",
+      currentUser: ""
+    };
+  }
+  componentWillMount() {
+    this.fetchLocation('/location').then((response) => {
+      this.setState({locationTags: response.results});
+      this.setState({defaultLocation: response.results[0].display});
+    });
 
-        this.fetchLocation('/session').then((response) => {
-            this.setState({currentUser: response.user.display});
-        });
-    }
+    this.fetchLocation('/session').then((response) => {
+      this.setState({currentUser: response.user.display});
+    });
+  }
 
-    getLocations() {
-        return this.state.locationTags.map((location) => {
-            return location.display;
-        });
-    }
+  getLocations() {
+    return this.state.locationTags.map((location) => {
+      return location.display;
+    });
+  }
 
-    fetchLocation(url) {
-        const apiHelper = new ApiHelper(null);
-        const getData = new Promise(function(resolve, reject) {
-            apiHelper.get(url).then(response => {
-                response.json().then(data => {
-                    resolve(data);
-                });
-            });
+  fetchLocation(url) {
+    const apiHelper = new ApiHelper(null);
+    const getData = new Promise(function(resolve, reject) {
+      apiHelper.get(url).then(response => {
+        response.json().then(data => {
+          resolve(data);
         });
-        return getData;
-    }
+      });
+    });
+    return getData;
+  }
 
-    handleClick(e) {
-        e.preventDefault();
-        this.setState({currentLocationTag: e.target.id});
-    }
+  handleClick(e) {
+    e.preventDefault();
+    this.setState({currentLocationTag: e.target.id});
+  }
 
-    dropDownMenu(locationTags) {
-        const menuDisplay = [];
-        const numPerColumn = Math.ceil(locationTags.length / NUMBER_OF_COLUMNS);
-        for (let cols = 0; cols < NUMBER_OF_COLUMNS; cols++) {
-            const menuInColumns = [];
-            let colStart = cols * numPerColumn;
-            let colEnd = (cols + 1) * numPerColumn;
-            for (let menuIndex = colStart; menuIndex < colEnd; menuIndex++) {
-                menuInColumns.push(
+  dropDownMenu(locationTags) {
+    const menuDisplay = [];
+    const numPerColumn = Math.ceil(locationTags.length / NUMBER_OF_COLUMNS);
+    for (let cols = 0; cols < NUMBER_OF_COLUMNS; cols++) {
+      const menuInColumns = [];
+      let colStart = cols * numPerColumn;
+      let colEnd = (cols + 1) * numPerColumn;
+      for (let menuIndex = colStart; menuIndex < colEnd; menuIndex++) {
+        menuInColumns.push(
                     <a href="#" key={menuIndex} id={locationTags[menuIndex]} onClick={(e) => {
```

Comment:
No they don't, @paulupendo, However this PR focuses on just fixing the indentation issues since they are everywhere, a new ticket would be opened to fix every other EsLint issue we have found so far.